duckdns/README.md

```diff
@@ -36,20 +43,49 @@ Add-on configuration:
 }
 ```
 
-### Option: `lets_encrypt`
+### Option group `lets_encrypt`
+
+---
+
+The following options are for the option group: `lets_encrypt`. These settings
+only apply to Let's Encrypt SSL  certificates.
+
+
+#### Option `lets_encrypt`: `accept_terms`
```

Comment:
Let us make it consistent with how we handle services on our documentation website.

```suggestion
### Option `lets_encrypt.accept_terms`
```